src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -31,6 +37,26 @@ const formConfig = {
           path: 'form-page',
           title: 'First Page',
           uiSchema: {
+            name: _.merge(autoSuggestUiSchema(
+              'Search for some schools',
+              value => fetchAutocompleteSuggestions(value)
+                .then(transformAutocompletePayloadForAutosuggestField),
+              {
+                'ui:options': {
+                  queryForResults: true,
+                  freeInput: true,
```

Comment:
I'm not sure if this should be true or false- it's used in a [couple](https://github.com/usds/us-forms-system/blob/0fc545d47a10c530ee6b944a7040853e828c3d94/src/js/fields/AutosuggestField.jsx#L132) of places to determine if the data passed to the `onChange` prop needs to have a `widget` property. It causes the new component to throw this error when it's false:

![screen shot 2018-07-19 at 08 53 18](https://user-images.githubusercontent.com/4998130/42950432-3c2d87c2-8b31-11e8-8ca8-de881ed5ee80.png)